infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -243,20 +253,33 @@ private void generateLogicalGroupConfig(LogReplicationSession session) {
                     logicalGroupToStreams.put(logicalGroup, relatedStreams);
                     // Collect tags for this stream
                     List<UUID> tags = streamToTagsMap.getOrDefault(streamId, new ArrayList<>());
-                    tags.addAll(entry.getValue().getMetadata().getTableOptions().getStreamTagList().stream()
```

Comment:
Commenting here as I am unable to add a comment on the unchanged code in L242.  LOG_REPLICATOR_STREAM must not be put as the tag here, right?

Reply:
yes good catch! it should be the corresponding opaque stream tag that derives from client name